libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/offroute/OffRouteDetector.java

```diff
@@ -52,10 +51,13 @@
   @Override
   public boolean isUserOffRoute(Location location, RouteProgress routeProgress, MapboxNavigationOptions options) {
 
+    if (checkDistanceRemaining(routeProgress)) {
```

Comment:
We should reflect this new check in the Javadoc ☝️

Reply:
It'd be great if we add a test in `OffRouteDetectorTest` around this new check.

Reply:
Both of these have been addressed 👍